### PR TITLE
Four lenses of productivity

### PR DESCRIPTION
**Pull request by sapanparikh18**
## What

Please include a summary of the changes.

- [x] Added item
- [ ] Updated item

## How

Create a new blog

### CODE REVIEW

File: content/blog/four-lenses-of-productivity/ind-team-org-market.json
Comment (2KAbhishek) [inline content/blog/four-lenses-of-productivity/ind-team-org-market.json:1]: small nit: minifying this file will save us ~60kb of bandwidth https://codebeautify.org/jsonminifier

File: content/blog/four-lenses-of-productivity.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+date = 2023-07-24T00:00:00Z
+description = "How productivity varies at individual, teams, organization and market leve and what can impact it."
+draft = false
+image = "/images/2021/11/checklist2.jpg"
+slug = "four-lenses-of-productivity"
+tags = ["DevEx"]
+title = "Four Lenses of Productivity: Incubyte Take"
++++
+
+{{< circlepack data="ind-team-org-market.csv" >}}
```
Comment (2KAbhishek) [inline content/blog/four-lenses-of-productivity.md:12]: The data param says csv, but the file we have is a json: `ind-team-org-market.json`
Is that intentional?

File: content/blog/four-lenses-of-productivity.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+## Team Productivity:
+> When we think about team productivity, we think about how efficiently the team can meet requirements.
+
+A team’s productivity is not equal to the sum of every individual’s productivity. It may not even be the average of everyone’s productivity. It may be as high as the slowest bottleneck of the value stream!
+
+### Breaks
+Work queues decrease a team’s productivity. These queues are usually created at the tail end of the software delivery process, like in manual testing. Also, having different definitions of done for each team role makes everyone busy (in moving tickets to the next guy) but may not make the team productive.
+
+### Fule
```
Comment (2KAbhishek) [inline content/blog/four-lenses-of-productivity.md:39]: Small typo
```suggestion
### Fuel
```

**Comment by akshayvadher** [comment]
The graph view looks good and the blog overall looks good as well.

**Review by swathynair19 — APPROVED** [review]

**Review by swathynair19 — APPROVED** [review]

**Review by swathynair19 — APPROVED** [review]

**Pull request merged**